# from pr2_teleop/config/
# adapted for mitro

# Axes
axis_vx: 1
axis_vy: 0
axis_vw: 0

# base velocities
max_vw: 1.25
max_vx: 0.5
max_vy: 0.0
max_vw_run: 2.0
max_vx_run: 1.0
max_vy_run: 0.0

run_button: 11
deadman_button: 10
